unittests/Parse/LexerTests.cpp

```diff
@@ -277,10 +277,10 @@ TEST_F(LexerTest, BOMNoCommentNoTrivia) {
   LangOptions LangOpts;
   SourceManager SourceMgr;
   unsigned BufferID = SourceMgr.addMemBufferCopy(StringRef(Source));
-  
+
   Lexer L(LangOpts, SourceMgr, BufferID, /*Diags=*/nullptr, LexerMode::Swift,
-          HashbangMode::Disallowed, CommentRetentionMode::None);
-  
+          HashbangMode::Disallowed, CommentRetentionMode::AttachToNextToken);
```

Comment:
This was specially testing `CommentRetentionMode::None` behavior. Removing the case.